src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { TodoItem } from '../TodoItem';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[]
+}
+
+export const TodoList: React.FC<Props> = React.memo(
+  ({ todos }) => {
+    return (
```

Comment:
if there is no logic in component and you have only jsx, then you can return without using word `return`

`React.memo(({ todos }) => (`